src/controllers/events/modify_content.js

```diff
@@ -0,0 +1,70 @@
+const Request = require('request');
+const { eventsURL } = require('../../constants/urls.json');
+
+module.exports = (req, res) => {
+  const apiBody = {
+    imageUrl: req.body.imageUrl,
+    cost: req.body.cost,
+    startTime: req.body.startDate,
+    endTime: req.body.endDate,
+    place: req.body.eventPlace
+  };
+
+  if (req.body.name_en) {
+    apiBody.en = {
+      name: req.body.name_en,
+      description: req.body.description_en
+    };
+  }
+
+  if (req.body.name_ar) {
+    apiBody.ar = {
+      name: req.body.name_ar,
+      description: req.body.description_ar
+    };
+  }
+
+  apiBody.categories = req.body.categories;
+  if (req.body.accessibility) apiBody.accessibilityOptions = req.body.accessibility;
```

Comment:
So at the moment if I edit an event and want to remove all accessibility options it won't work, as this line won't run - I think it needs to be:
`apiBody.accessibilityOptions = req.body.accesbilityOptions || null`

Reply:
In fact it could go like that at the end of your [apiBody definition here](https://github.com/foundersandcoders/OTP-Data-Entry/pull/56/files#diff-a4d7da68cfdf96ddc5bc1fa9ae2b0faaR10)